bb8/src/internals.rs

```diff
@@ -290,3 +305,8 @@ pub(crate) enum StatsGetKind {
     Waited,
     TimedOut,
 }
+
+pub(crate) enum StatsConnectionClosedKind {
```

Comment:
Since there's nothing "special" about these values, let's just call this `StatsKind` again.

Reply:
TBH i would prefer to keep the ConnectionClosed since this is used for recording closed connections, but Ill make the change

Reply:
It can be used for any kind of simple increment of `AtomicU64`.

Reply:
Its ok, lets see how `record` evolves .... BTW added the changes that you asked

Reply:
If we rename the enum might also rename the variants to be like `ClosedBroken`, `ClosedInvalid`.